### PR TITLE
Fixed #33724 -- Doc'd exclude argument changes in model validation.

### PR DESCRIPTION
**Pull request by felixxm**
Thanks אורי for the report.

Follow up to 1ea7e3157d1f9b4db71e768d75ea57e47dbd49f9.

ticket-33724

### CODE REVIEW

**Comment by felixxm** [comment]
buildbot, test this please.

**Review by claudep — APPROVED** [review]
The patch is fine, thanks. I still have mixed feelings about that `set` change. Now we have Model.exclude that should be a `set`, but `ModelForm.exclude`/`ModelAdmin.exclude` that should still be a list? Is that right?

**Comment by claudep** [comment]
Feel free to commit, thanks for your efforts!

File: docs/ref/models/instances.txt [hunk 1]
```diff
@@ -249,12 +249,16 @@ The first step ``full_clean()`` performs is to clean each individual field.
 
     The ``validate_constraints`` argument was added.
 
+.. versionchanged:: 4.1
+
+    An ``exclude`` value is now converted to the ``set`` instead of ``list``.
```
Comment (carltongibson) [inline docs/ref/models/instances.txt:254]:
```suggestion
    An ``exclude`` value is now converted to a ``set`` rather than a ``list``.
```

**Review by carltongibson — APPROVED** [review]

File: docs/ref/models/instances.txt (hunk 1 above)
Comment (felixxm) [reply docs/ref/models/instances.txt:254]: Updated :+1:

**Pull request merged**